src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.CreateToken.cs

```diff
@@ -35,14 +35,7 @@ public virtual string CreateToken(string payload)
         {
             _ = payload ?? throw LogHelper.LogArgumentNullException(nameof(payload));
 
-            return CreateToken(
-                    payload,
-                    null,
-                    null,
-                    null,
-                    null,
-                    null,
-                    null);
+            return CreateToken(payload, new SecurityTokenDescriptor());
```

Comment:
SecurityTokenDescriptor can be a static instance in this case to save the allocation.